resources/qml/Preferences/GeneralPage.qml

```diff
@@ -355,6 +357,20 @@ UM.PreferencesPage
             }
 
             UM.TooltipArea {
+                width: childrenRect.width;
+                height: childrenRect.height;
+                text: catalog.i18nc("@info:tooltip", "Should zooming move in the direction of the mouse?")
+
+                CheckBox
+                {
+                    id: zoomToMouseCheckbox
+                    text: catalog.i18nc("@action:button","Zoom toward mouse direction");
```

Comment:
Code style: Space after comma. I've fixed that for you here: ef89fab1269cc595fda15ea03d9760789efc3d29